web/shared/src/poll_data.ts

```diff
@@ -162,41 +235,38 @@ export class PollData {
             },
         };
 
-        if (question) {
-            this.set_question(question);
-        }
-
         for (const [i, option] of options.entries()) {
             this.handle.new_option.inbound("canned", {
```

Comment:
Looks like "canned" is arbitrary. If this is where the `string` `sender_id` comes from, we may as well change it to something that is compliant with the type.

Reply:
Yeah agreed! Should I change it to maybe -1?